server/src/routes/clientRouter.js

```diff
@@ -3,9 +3,20 @@ import getClientIndexPath from "../config/getClientIndexPath.js";
 
 const router = new express.Router();
 
-const clientRoutes = ["/", "/user-sessions/new", "/users/new"];
+const clientRoutes = ["/", "/user-sessions/new", "/users/new", "/movies", "/movies/:id/new", "/movies/:id"];
```

Comment:
should only have routes defined that are being used by this feature (there is not page corresponding to routes `"/movies/:id/new", "/movies/:id"`